payload-examples/README.md

```diff
@@ -1,4 +1,266 @@
-# Payload examples
+# Octokit Webhooks
+
+> machine-readable, always up-to-date GitHub Webhooks specifications
+
+![Update status](https://github.com/octokit/webhooks/workflows/Update/badge.svg)
+
+## Download
+
+Download the latest specification at
+[octokit.github.io/webhooks/index.json](https://octokit.github.io/webhooks/index.json)
```

Comment:
I think this will now be 

```suggestion
[octokit.github.io/webhooks/payload-examples/index.json](https://octokit.github.io/webhooks/payload-examples/index.json)
```

Reply:
I also found another place that needed to have the URL updated.

Committed here: https://github.com/octokit/webhooks/pull/436/commits/8c9dee4da0e441fb1cd93c4704eb4c5399597998